quora-service/src/main/java/com/upgrad/quora/service/dao/UserDao.java

```diff
@@ -1,6 +1,7 @@
 package com.upgrad.quora.service.dao;
 
-import com.upgrad.quora.service.entity.UserAuthEntity;
+//import com.upgrad.quora.service.entity.UserAuthEntity;
```

Comment:
rather than commenting an import, i would say remove it.

Reply:
Done.